assets/js/theme/common/product-details.js

```diff
@@ -17,11 +17,22 @@ export default class ProductDetails extends ProductDetailsBase {
         this.imageGallery = new ImageGallery($('[data-image-gallery]', this.$scope));
         this.imageGallery.init();
         this.listenQuantityChange();
+        this.swatchOptionMessageText = $('.swatch-option-message').text();
 
         const $form = $('form[data-cart-item-add]', $scope);
         const $productOptionsElement = $('[data-product-option-change]', $form);
         const hasOptions = $productOptionsElement.html().trim().length;
         const hasDefaultOptions = $productOptionsElement.find('[data-default]').length;
+        const $productSwatchGroup = $('[id*="attribute_swatch"]', $form);
+
+        if (context.showSwatchNames) {
+            $('.swatch-option-message').removeClass('u-hidden');
```

Comment:
Could we move $('.swatch-option-message') to class variable, we are using it at least 3 times

Reply:
done